person.rb

```diff
@@ -1,12 +1,13 @@
 class Person
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
Great job so far. Please add a method to create `Rentals` since  A`Person` has many `Rentals`. It is also one of the project requirements. 
![image](https://github.com/lerfast/opp-school-library/assets/101523582/02dc69b1-7208-4c6c-a32e-ac4a7485d139)

You can visit this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more. Kindly add it.

Reply:
![image](https://github.com/lerfast/opp-school-library/assets/111823606/a4d07a67-e8c1-4cc2-bfde-e37679c647a1)
Done!!!